includes/Story_Post_Type.php

```diff
@@ -552,6 +570,10 @@ public function get_editor_settings() {
 				'metadata'              => [
 					'publisher' => $this->get_publisher_data(),
 				],
+				'postLock'              => [
+					'interval' => $time_window,
+					'enabled'  => $show_locked_dialog,
```

Comment:
As per my other comment, something like `'showLockedDialog' => $show_locked_dialog` seems more clear

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/6497/commits/fcacf1e30d2c9e95b7700fc326651fa756cad34c